### ISSUE TITLE
Automatically rebuild on dependabot-preview PRs

### ISSUE THREAD

**Issue by rullzer** [skjnldsv/npmbuildbot#24]
For the use case we use this it would be very useful I think to also listen to the PR create hook.

If the PR is created by dependabot-preview we just do the compile step directly.

**Comment by MichaIng** [comment]
Good idea. This reduces manual efforts and overhead for drone and check servers. Not sure if it's possible to skip checks for the initial dependabot PR and instead wait for the npmbuildbot amend, or to have npmbuildbot amend to the dependabot branch, before the PR is even opened?
